crates/fuel-core/src/database.rs

```diff
@@ -90,7 +90,10 @@ pub struct Database<Description = OnChain>
 where
     Description: DatabaseDescription,
 {
+    /// Cached value from Metadata table, used to speed up lookups.
```

Comment:
See https://github.com/FuelLabs/fuel-core/pull/1694/commits/7b1141ac838568e3590f09dd420cb24a6946bd32#r1504911777 for the explanation behind this comment